policy/predicate/file.go

```diff
@@ -141,6 +141,44 @@ func (pred *OnlyChangedFiles) Trigger() common.Trigger {
 	return common.TriggerCommit
 }
 
+type NoChangedFiles struct {
+	Paths       []common.Regexp `yaml:"paths"`
+	IgnorePaths []common.Regexp `yaml:"ignore"`
+}
+
+var _ Predicate = &NoChangedFiles{}
+
+func (pred *NoChangedFiles) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
+	changedFiles := ChangedFiles{
+		Paths:       pred.Paths,
+		IgnorePaths: pred.IgnorePaths,
+	}
+
+	changedFilesPredicateResult, err := changedFiles.Evaluate(ctx, prctx)
+	if err != nil {
+		return nil, err
+	}
+
+	predicateResult := common.PredicateResult{
+		Values:          changedFilesPredicateResult.Values,
+		Description:     changedFilesPredicateResult.Description,
+		Satisfied:       !changedFilesPredicateResult.Satisfied,
+		ValuePhrase:     "changed files",
+		ConditionPhrase: "cannot match",
+		ConditionsMap:   changedFilesPredicateResult.ConditionsMap,
```

Comment:
The way phrases are used is pretty awkward for negative conditions: when the predicate is skipped the `ConditionPhrase` appears after `do not`. In this case that leads to "The changed files [...] do not cannot match path patterns ..."

The best single word I can think of right now is "avoid", leading to "The changed files [...] avoid path patterns ..." and "The changed files [...] do not avoid path patterns ...". This works, but is not great.

A more involved option is to modify the template to do something different here. Maybe if the `ConditionPhrase` value already has `do not` as a prefix it could remove it when the predicate is skipped instead of adding it.